src/CMakeLists.txt

```diff
@@ -1,28 +1,32 @@
-set(AKTUALIZR_LITE_SRC main.cc helpers.cc)
-set(AKTUALIZR_LITE_HEADERS helpers.h)
+set(TARGET aktualizr-lite)
+set(SRC main.cc helpers.cc)
+set(HEADERS helpers.h)
 
-add_executable(aktualizr-lite ${AKTUALIZR_LITE_SRC})
-target_link_libraries(aktualizr-lite aktualizr_lib)
+add_executable(${TARGET} ${SRC})
 
-install(TARGETS aktualizr-lite RUNTIME DESTINATION bin COMPONENT aktualizr-lite)
+if(CLANG_TIDY)
+set_target_properties(${TARGET}
+  PROPERTIES CXX_CLANG_TIDY
+# TODO: consider adding clang-tidy options that it makes sense to include here
+  clang-tidy-6.0;-checks=*;-warnings-as-errors=*
+)
```

Comment:
I assume this is the same list that aktualizr is using?

Reply:
Not exactly, that's what aktualizr does https://github.com/advancedtelematic/aktualizr/blob/9b0ce2144c8d4630aa7ee3fcca2c603df493bb56/CMakeLists.txt#L278. I don't think that the diff is critical.
The given approach cleaner as it makes use of the make built-in functionality while the aktualizr implements their own functionality and introduces a custom target for each source file. In terms of clang-tidy options, we can put there whatever we consider it makes sense to do.